src/main/java/frc/robot/hardware/mechanisms/SimulatedAccelerometer.java

```diff
@@ -0,0 +1,70 @@
+package frc.robot.hardware.mechanisms;
+
+import edu.wpi.first.math.kinematics.ChassisSpeeds;
+import frc.robot.hardware.interfaces.IAccelerometer;
+import frc.utils.alerts.Alert;
+import frc.utils.alerts.AlertManager;
+import frc.utils.alerts.PeriodicAlert;
+import frc.utils.time.TimeUtils;
+
+import java.util.Random;
+import java.util.function.Supplier;
+
+public class SimulatedAccelerometer implements IAccelerometer {
+
+	private final Supplier<ChassisSpeeds> robotSimulatedVelocityRelativeToRobot;
+	private final Supplier<Double> noise;
```

Comment:
Consider changing noise to fakeNoise to make it more clear
```suggestion
	private final Supplier<Double> fakeNoise;
```